### DISCUSSION TITLE
Feature Request: Per Item Input Prompt size

### DISCUSSION THREAD

**Discussion by SoZoYo** [chhoumann/quickadd#606] [Ideas]
For some items, I want to enter several paragraphs. For others, it's just a word or two. Having a large input box for one word is awkward. Having a single line to enter several paragraphs is impossible. So for now I live with awkwardness. It would be great if we could set the input box size per item. Perhaps a default setting and an override option? 🙏

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

Exactly this shipped in QuickAdd 2.10.0: you can set the input size per item.

```
{{VALUE:Body|type:multiline}}
```

gives that one prompt a large textarea, while a plain `{{VALUE:Body}}` follows the global default (**Use Multi-line Input Prompt** in QuickAdd settings). So it works the way you asked for: a default setting plus a per-item override.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#value-multiline

Closing as resolved.
